examples/03-Running-on-multiple-GPUs-or-on-CPU.ipynb

```diff
@@ -0,0 +1,742 @@
+{
```

Comment:
<p>can you add the to_parquet command?</p><p> </p><p>I think that is the default NVTabular workflow, we want to teach our users.</p><p>We are using small datasets for our examples, but the library is designed to scale to 100GBs. Although we use small datasets, we should write the examples that they would work on large dataset. Users often copy the examples and just adjust them</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/NVTabular/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
done